Robot2019/src/main/java/frc/robot/commands/TeleopDrive.java

```diff
@@ -7,12 +7,16 @@
 
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.command.Command;
 import frc.robot.subsystems.Drivetrain;
 
 public class TeleopDrive extends Command {
+  Drivetrain dt;
+
   public TeleopDrive(Drivetrain dt) {
```

Comment:
You should pass in the Joystick here, and run the logic here. To run the motors, call setter functions in drivetrain.